base/pkg/dir.jl

```diff
@@ -35,15 +35,25 @@ function init(meta::AbstractString=DEFAULT_META, branch::AbstractString=META_BRA
     end
     dir = path()
     info("Initializing package repository $dir")
+    temp_dir = joinpath(dir, "TEMP")
     metadata_dir = joinpath(dir, "METADATA")
+    #Check if TEMP dir exists from previous failed init
+    if isdir(temp_dir)
+      rm(temp_dir, recursive=true)
+      #If METADATA still exists then an incomplete oopy may have occured
+      #Or temp was not fully deleted
+      #Remove METADATA as well and reinit rather
+      if isdir(metadata_dir)
+        rm(metadata_dir, recursive=true)
```

Comment:
Still learning git, bad copy I forgot to check after doing reset >_< Will fix that now.